adal/src/main/java/com/microsoft/aad/adal/AuthenticationRequest.java

```diff
@@ -187,6 +189,26 @@
         mIsExtendedLifetimeEnabled = isExtendedLifetimeEnabled;
     }
 
+    /**
+     * Create an Authentication Request to obtain a token. This overload also accepts a scope param.
+     *
+     * @param authority                 Authority URL
+     * @param resource                  Resource that is requested
+     * @param clientId                  ClientId for the app
+     * @param correlationId             Correlation Id used for logging & telemetry
+     * @param isExtendedLifetimeEnabled a boolean indicating if extended lifetime enabled
+     * @param scope                     the scope requested
+     */
+    AuthenticationRequest(String authority, String resource, String clientId,
```

Comment:
There's lots of tech debt around nullability annotations in ADAL.... since we're adding new API here, can we add it to this method?

Reply:
Addressed